apps/native-component-list/src/screens/FirebasePhoneAuthScreen.tsx

```diff
@@ -0,0 +1,106 @@
+import { FirebaseRecaptchaVerifierModal } from 'expo-firebase-recaptcha';
+import { getApp, initializeApp } from 'firebase/app';
+import { getAuth, signInWithCredential, PhoneAuthProvider, UserCredential } from 'firebase/auth';
+import * as React from 'react';
+import { Alert, ScrollView, TextInput, LogBox } from 'react-native';
+
+import HeadingText from '../components/HeadingText';
+import ListButton from '../components/ListButton';
+import { firebaseConfig } from './FirebaseRecaptchaScreen';
+
+// See: https://github.com/firebase/firebase-js-sdk/issues/1847
+LogBox.ignoreLogs(['AsyncStorage has been extracted']);
+
+try {
+  // Try to initialize, if it doesn't work its already registered
+  initializeApp(firebaseConfig);
```

Comment:
iirc there was a system for detecting how many apps were initialized something like `if (firebase.apps.length)`

```suggestion
  initializeApp(firebaseConfig);
```